lib/mutant/mutator/node/regopt.rb

```diff
@@ -0,0 +1,31 @@
+module Mutant
+  class Mutator
+    class Node
+
+      # Regular expression options mutation
+      class Regopt < self
+
+        MUTATED_FLAGS = IceNine.deep_freeze([:i])
```

Comment:
Lets use `%i[]` here? AFAIK mutant does not support rubies where this syntax does not exist.